### DISCUSSION TITLE
How use React Native Paper theme with styled components?

### DISCUSSION THREAD

**Discussion by gustavohbeg** [callstack/react-native-paper#3516] [General]
![image](https://user-images.githubusercontent.com/82586205/206339919-7a1abbb6-dc9f-43ef-a9d8-cfd1f707dd43.png)
I'm trying to use react native paper with styled components but I get the following error, can anyone answer?

**Comment by lukewalczak** [comment]
The issue here is wrongly created `theme`, the following snack presents how to handle it properly: https://snack.expo.dev/@react-native-paper/styledcomponents
